tests/damage_rheology_strain.prm

```diff
@@ -0,0 +1,134 @@
+# A test for the damage rheology material model. The test produces nearly 
+# constant temperatures, pressures, strain rates and uses experimentally 
+# derived values for olivine. Therefore the grain size in the test approaches
+# the equilibrium grain size for t -> infinity
```

Comment:
Can we write down what that value should be?

Reply:
Just for the record, I run the test until t = 4e6 years, and got a maximum grain size of 9.35850647e-04 (which was the equilibrium value). Could you add those number to the .prm file? Otherwise I think this test is fine.